examples/3DTilesPointCloud.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>3D Tiles Point Cloud example</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/default.css" />
+  </head>
+  <body>
+    <script src="../packages/browser/dist/RUN_MODE/bundle.js"></script>
+
+    <script type="text/javascript">
+      udvizBrowser.FileUtil.loadMultipleJSON([
+        './assets/config/extent_lyon.json',
+        './assets/config/frame3D_planars.json',
+        './assets/config/layer/3DTiles_point_cloud.json',
+        './assets/config/layer/base_maps.json'
+      ]).then((configs) => {
+        // http://proj4js.org/
+        // define a projection as a string and reference it that way
+        // the definition of the projection should be in config TODO_ISSUE
+        udvizBrowser.proj4.default.defs(
+          'EPSG:3946',
+          '+proj=lcc +lat_1=45.25 +lat_2=46.75' +
+            ' +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs'
+        );
+
+        const extent = new udvizBrowser.itowns.Extent(
+          configs['extent_lyon'].crs,
+          parseInt(configs['extent_lyon'].west),
+          parseInt(configs['extent_lyon'].east),
+          parseInt(configs['extent_lyon'].south),
+          parseInt(configs['extent_lyon'].north)
+        );
+
+        const frame3DPlanar = new udvizBrowser.Frame3DPlanar(
+          extent,
+          configs['frame3D_planars'][3]
+        );
+
+        udvizBrowser.THREEUtil.addLights(frame3DPlanar.getScene());
+        udvizBrowser.THREEUtil.initRenderer(
+          frame3DPlanar.getRenderer(),
+          new udvizBrowser.THREE.Color(0x6699cc)
+        );
+
+        // ADD BASE MAP
+        udvizBrowser.addBaseMapLayer(
+          configs['base_maps'][0],
+          frame3DPlanar.itownsView,
+          extent
+        );
+
+        // /// ADD LAYER
+        udvizBrowser.add3DTilesLayers(
```

Comment:
I know there is a also [3DTiles point cloud](https://github.com/CesiumGS/3d-tiles/tree/main/specification#introduction) but what is the difference ?

Reply:
PointCloudLayer is used to load either [Potree](https://github.com/iTowns/itowns/blob/6953e0119c35a550621aa792a204c352731aca97/src/Layer/PotreeLayer.js) point cloud or [Entwine](https://github.com/iTowns/itowns/blob/6953e0119c35a550621aa792a204c352731aca97/src/Layer/EntwinePointTileLayer.js) point cloud. 
Since 3DTiles handle both mesh and point cloud, and this example is about a 3DTiles point cloud, I think used the appropriate layer.

Reply:
Thank you for clarifying me :)